dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/model/OkHttpRequestHeaderContentType.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to Apache Software Foundation (ASF) under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Apache Software Foundation (ASF) licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.dolphinscheduler.common.model;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum OkHttpRequestHeaderContentType {
+
+    APPLICATION_JSON("application/json"),
+    APPLICATION_FORM_URLENCODED("application/x-www-form-urlencoded");
```

Comment:
Why not use `org.apache.http.entity.ContentType`, which contains more complete enum.

Reply:
There are some types in `org.apache.http.entity.ContentType` that we do not currently support, or are not sure if we support, we need to check through a configuration to use the default `application/json` if not supported. This is also described in the documentation to inform the users.